StanwoodCore/Core/Storage.swift

```
@@ -80,7 +96,7 @@ extension Stanwood {
             } catch {
                 
```

This should throw an error.